doc/examples/commctl_host_create.rst

```diff
@@ -1,4 +1,4 @@
 .. code-block:: shell
 
-   $ commctl host create 192.168.152.110 /path/to/priv/ssh_key
+   $ commctl host create --cluster my_cluster 192.168.152.110 /path/to/priv/ssh_key
```

Comment:
wait. below in `Manual Registration` it says you need the hosts admin users ssh key. Maybe belay my last comment.

But in the docs above I see the change in the usage string

> ``ssh_priv_key``: The full path to the **hosts** ssh private key for access

So is it the hosts private key or the admin users private key?